dmidecode ONIE cross compile patch

Copyright (C) 2017 Curt Brune <curt@cumulusnetworks.com>

SPDX-License-Identifier:     GPL-2.0

Small modifications to the dmidecode Makefile for cross compiling in
the ONIE build environment.
---
 Makefile |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 1f54a1f..1d03628 100644
--- a/Makefile
+++ b/Makefile
@@ -12,7 +12,7 @@
 #   (at your option) any later version.
 #
 
-CC      = gcc
+CC      = $(CROSS_COMPILE)gcc
 CFLAGS  = -W -Wall -Wshadow -Wstrict-prototypes -Wpointer-arith -Wcast-qual \
           -Wcast-align -Wwrite-strings -Wmissing-prototypes -Winline -Wundef
 
@@ -29,6 +29,9 @@ CFLAGS += -O2
 # Pass linker flags here
 LDFLAGS =
 
++CFLAGS		+= $(ONIE_CFLAGS)
++LDFLAGS	+= $(ONIE_LDFLAGS)
+
 DESTDIR =
 prefix  = /usr/local
 sbindir = $(prefix)/sbin
